Automattic/newspack-network
# feat: display membership plans across the network

### All Submissions:






### Changes proposed in this Pull Request:



Displays membership plans from the whole network:

<img width="1180" alt="image" src="https://github.com/Automattic/newspack-network/assets/7383192/72eeae27-12c1-4be6-a346-ccb2b4ac1407">

### How to test the changes in this Pull Request:

1. On Hub and Node site, set up some membership plans and link some of them via the "Network ID" feature (#40)
2. Visit the "Membership Plans" view in the Hub, observe that all memberships across the network are listed there
3. Make a change to the "Network ID" of a membership on a Node site, then click "Refetch Plans" on this view
4. Click "Edit" links and verify they point to the correct views
5. Observe the data has been updated accordingly

### Other information:
